src/ONYXKEYS.js

```diff
@@ -115,6 +115,9 @@ export default {
     // Stores information about additional details form entry
     WALLET_ADDITIONAL_DETAILS: 'walletAdditionalDetails',
 
+    // Stores values put into the additional details step of the wallet KYC flow
+    WALLET_ADDITIONAL_DETAILS_DRAFT: 'walletAdditionalDetailsDraft',
```

Comment:
I'm following the pattern we initially set up here:

https://github.com/Expensify/App/blob/1c80150632c684174270c019b7439854318471a4/src/ONYXKEYS.js#L133-L134

And yeah they are drafts since if the user bails on the flow and returns later they can pick up where they left off.

Reply:
This hopefully will be a detail of all forms eventually but for now we have to store it somewhere.